File: src/game.lua
local function game()
   math.randomseed(os.time())

   local min = 1
   local max = 100
   
   local random_num = math.random(min, max)
   local tries = 0

   print("Please enter a number between 1 - 100")

   while true do
	  local player_num = io.read("*number")

	  if player_num > random_num then
		 print("Number is too high")
		 tries = tries + 1
	  elseif player_num < random_num then
		 print("Number is too low")
		 tries = tries + 1
	  else
		 print("YOU HAVE WON!!!!!!!\nTries: " .. tries .. "\nNumber: " .. random_num .. "\n")
		 return 0
	  end
   end
end

game()

File: src/game.py
import random

def GenRandInt(Min: int, Max: int) -> int:
    return random.randint(Min, Max)

def main():
    random_int = GenRandInt(1, 100)
    tries: int = 0

    while True:
        player_int = int(input("Please enter a number betweenn 1 - 100: "))

        if player_int > random_int:
            print("Number is too high")
            tries += 1
        elif player_int < random_int:
            print("Number is too low")
            tries += 1
        elif player_int == random_int:
            print(f"YOU WIN!!!!!!!!!!!!!!!\nNumber: {random_int}\nTries: {tries}")
            exit()

main()
